src/EditorFeatures/CSharpTest/Completion/CompletionProviders/PropertySubPatternCompletionProviderTests.cs

```diff
@@ -503,7 +503,7 @@ class D
     public int P2 { get; set; }
 }
 ";
-            await VerifyNoItemsExistAsync(markup);
+            await VerifyItemExistsAsync(markup, "P1");
```

Comment:
Just trying to better understand, why was this file changed? Does it have to do with the `when` keyword or is it something separate?

Reply:
this used to be illegal in C# but became legal in C# 9.0. i.e. you can continually nest recursive patterns within each other :)